src/main.py

```diff
@@ -21,9 +21,8 @@
 gi.require_version("Gdk", "3.0")
 gi.require_version("Gtk", "3.0")
 gi.require_version("Gio", "2.0")
-gi.require_version("Handy", "1")
```

Comment:
We will still this one, because the app will crash if there are two version of libhandy on the system (when using non-flatpak version).

Reply:
ah I see